### ISSUE TITLE
UX statistics links behavior is not consistent with the header behavior

### ISSUE THREAD

**Issue by abollini** [DSpace/dspace-angular#1392]
**Describe the bug**
The statistics link in the header navbar is contextualized to the DSpace Object that is currently visualized, site, community, collection or item. This is confusing for end users, especially users that are unfamiliar with DSpace (that could be the majority as statistics is by default available to anonymous users).

In https://github.com/DSpace/dspace-angular/issues/695 a similar contextual behavior is suggested also for the search but again this would be confusing for the users as the top level menu of a website are usually intended to offer a first level of navigation and it is not expected to be contextual.

Moreover, some contextual behavior already exists at the top of the dspace objects page and it is currently used to show a  button providing quick access to the edit feature.

**To Reproduce**
1. Visit the demo home page https://demo7.dspace.org/home , check the value of the statistics link https://demo7.dspace.org/statistics
2. Visit a community, collection or item page, for instance https://demo7.dspace.org/items/08cb98b3-faff-4682-8a7b-010778ccc600 check the value of the statistics link https://demo7.dspace.org/statistics/items/08cb98b3-faff-4682-8a7b-010778ccc600

in addition the statistics link is placed in different place of the navbar in the home page (last position) than in the item page (first position)

**Related work**
https://github.com/DSpace/dspace-angular/issues/695

**Comment by tdonohue** [comment]
@abollini : I agree with this.  Perhaps it's better to consider just moving the statistics link _into the body of the page itself_, rather than having it be a part of the header.  Having the link in the header make it appear more as a "global" (not context specific) option. Whereas, if it were in the body of the page, it might be a separate button or tab that brought you to the statistics for that specific page.

That said, I don't like the direction suggested by #695, as I worry that making the header options context specific can result in more confusing behavior.  As already noted in this ticket, the fact that that "Statistics" link is context-specific is confusing...so trying to make more of the header context-specific would not solve that issue.    I feel a better direction is to possibly ensure the header remains *global* (non-context-specific)....anything else might result in a very complex or messy header.

**Comment by tdonohue** [comment]
I'm bumping up the priority on this to `high priority`, as I've heard of similar confusion from several people now.  The current behavior of the "Statistics" header link is not ideal, and it may need some rethinking.

**Comment by artlowel** [comment]
An alternative, or perhaps workaround until we find something better may be to change the wording of the statistics link in the header as the context changes as well: "Site Statistics", "Statistics for this Collection", etc

Or turn statistics into a dropdown menu, and always keep all options based on the context: so on a homepage that would be

```
Statistics ▼ 
|- All of DSpace
```

And on an item page

```
Statistics ▼
|- All of DSpace
|- Communty A
|- Sub-community B
|- Collection C
|- Item D
```
